Source/santabs/SNTBundleService.m

```diff
@@ -231,6 +236,7 @@ - (NSArray *)findRelatedBinaries:(SNTStoredEvent *)event progress:(NSProgress *)
       dispatch_group_async(group,
                            dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0), ^{
         @autoreleasepool {
+          fi.useAncestorBundle = YES;
```

Comment:
I think there's an opportunity for an optimization here, particularly as we're making the bundle searching more time-consuming. Where you replace the bundle path on line 97, replace all the other bundle properties too, and then copy those to the new event below (lines 246-249).